pages/instructorAssessmentQuestionManualGrading/instructorAssessmentQuestionManualGrading.sql

```diff
@@ -0,0 +1,29 @@
+-- BLOCK select_question
+SELECT
+    q.id AS question_id,
+    q.title AS question_title,
+    aq.max_points
+FROM
+    assessment_questions AS aq
+    JOIN questions AS q ON (q.id = aq.question_id)
+WHERE
+    aq.assessment_id = $assessment_id
+    AND aq.id = $assessment_question_id;
+
+
+-- BLOCK select_instance_questions_manual_grading
```

Comment:
It looks to me like this is selecting all submissions for the given `assessment_question`, not all `instance_questions`. Is that what we want? If so, can we change the BLOCK name to reflect this?

But I'm a bit confused, because the JS and EJS files seem to be treating the returned data as a list of instance questions. What should it be?

Reply:
It is the last submission of a each specific instance question (see the DISTINCT ON at the top). So in essence it's selecting all instance questions, together with the last submission of that instance question.

Reply:
Oh right! Somehow I overlooked this. Sorry!